mesos-commons/src/main/java/org/apache/mesos/collections/Maps.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.collections;
+
+import com.google.common.base.Predicate;
+
+import javax.annotation.Nonnull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ */
+public class Maps {
+
```

Comment:
It wasn't intended to be... I can see common cases where there is a need to filter a map...  we could just allow users to do it themselves, however I felt the need to use this predicate would be common.  I could pull out just the predicate for reuse.   I expect having a start to a property as a common value to be common... like `MESOS` or `HDFS`